lib/rules/no-side-effects-in-computed-properties.js

```diff
@@ -31,15 +31,9 @@ module.exports = {
   create(context) {
     /** @type {Map<ObjectExpression, ComponentComputedProperty[]>} */
     const computedPropertiesMap = new Map()
-    /**
-     * @typedef {object} ScopeStack
-     * @property {ScopeStack} upper
-     * @property {BlockStatement | Expression} body
-     */
-    /**
-     * @type {ScopeStack}
-     */
-    let scopeStack
+
+    /** @type { { upper: any, body: null | BlockStatement | Expression } } */
```

Comment:
I referred to
https://github.com/vuejs/eslint-plugin-vue/blob/eb20820df997cbd3afef4ea2a3589f7a04ffb620/lib/rules/require-valid-default-prop.js#L109